src/fs/v1/PublicTree.ts

```diff
@@ -28,6 +33,7 @@ export class PublicTree extends BaseTree implements HeaderTree {
     }
   }
 
+  // Wait wait... why async?
   static async empty (parentKey: Maybe<string>): Promise<HeaderTree> {
```

Comment:
@dholms I'm unclear why an empty tree would need to be a promise. I realize that different classes will have different representations, but in all cases we would know the CID in advance, and it should always return a consistent structure, no?

Reply:
The `PrivateTree.empty` static method is async (because generating a new symm key through web crypto is async). So I made `PublicTree.empty` async as well so that the interface would be the same

Reply:
Ooooh I see. Gotcha thanks 👍 That's annoying 😛

Reply:
From a quick search — it sounds like the JS convention for this is `init`. I like `empty` because of the monoidal implications, but it _is_ running a side effect in the browser. Anyhow, naming is hard, right?

Reply:
yup :stuck_out_tongue: 
I chose `empty` because it was making it clear that the file didn't have any children (in opposition to `fromCID` or `fromHeader` which is pulling some info from another source so it might have children). But I'm fine with `init` as well